package.json

```diff
@@ -8,25 +8,23 @@
     "start": "next start"
   },
   "dependencies": {
-    "@chakra-ui/react": "^1.6.1",
+    "@chakra-ui/react": "^1.6.5",
```

Comment:
[Reference for updating all packages to their latests versions](https://nodejs.dev/learn/update-all-the-nodejs-dependencies-to-their-latest-version)